Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:7
+
+RUN yum update -y && \
+    yum install -y \
+    epel-release-7 \
+    zip \
+    unzip \
+    java-1.8.0-openjdk \
+    maven \
+    make && \
+    yum clean all
+
+COPY test-data-generator.jar /root/test-data-generator.jar
```

Comment:
Weren't we going to change the jar location to `/opt/<something>`?

Reply:
Yep, that's still to do. It's on the list... 👍

Reply:
Done. The jar now lives in **/opt/test-data-generator/** and the start script is in **/usr/local/bin/**